man/systemd.socket.xml

```diff
@@ -429,6 +429,17 @@
         false, in read-only mode. Defaults to false.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>FlushPending=</varname></term>
+        <listitem><para>Takes a boolean argument. May only be used when
+        <option>Accept=no</option>. If true, the socket's buffers are cleared after the
+        triggered service exited. The result will be that any pending data will be
```

Comment:
The result will be that ...  → This causes ...

Reply:
@keszybz BTW there are inconsistencies all between `man/org.freedesktop.systemd1.xml` and `man/systemd.socket.xml` for `Accept`: sometimes it's written `yes/no`, sometimes `on/off`, sometimes `true/false`.
Shall I fix this to have `yes/no` everywhere as part of this PR?

Reply:
It's not really an error, since all combinations are equivalent. It'd be nice to clean that up for consistency, but I think it should be a separate PR.